tox.ini

```diff
@@ -11,8 +11,8 @@ setenv =
     TEST_NET=ganache
     PYTHONPATH = {toxinidir}
     CONFIG_FILE = {toxinidir}/config.ini
-    TEST_PRIVATE_KEY1=0xbbfbee4961061d506ffbb11dfea64eba16355cbf1d9c29613126ba7fec0aed5d
-    TEST_PRIVATE_KEY2=0x804365e293b9fab9bd11bddd39082396d56d30779efbb3ffb0a6089027902c4a
+    TEST_PRIVATE_KEY1=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58
```

Comment:
Don't change private keys, please! We rely on these in various places.

Reply:
I talked with @trentmc to change those keys regarding to the web3 unit test.